docs/gameplay-sessions/gameplay-sessions-reference.mdx

```diff
@@ -16,13 +16,13 @@ When a Gameplay Session is recorded, all data is stored on disk within two folde
 directory (which we will reference as the `sessionDirectory`):
 
 ```
-~/unity_videos/{Application.productName}/{replay|recording}_{date}_{time}_{currentSessionId}_{referenceSessionId}
+~/unity_videos/{Application.productName}/{replay|recording}_{date}_{time}_{currentSessionId}_{referenceSessionId|currentSessionId}
```

Comment:
What I was trying to get at Aaron is that this isn't correct.

its either `..._{currentSessionId}` or `...{currentSessionId}_{referenceSessionId}`  ; we don't show the currentSessionId 2x on recordings

Reply:
Yes this will be fixed soon! I have no idea why, but I swear in one of my recordings, it had it twice... was it like that before? I deleted all of my recordings recently and now don't see that behavior. But yes that will be fixed soon! there are just some merge conflicts I need to address

Reply:
@vontell  Was it intentional that your merge wiped out these changes ?

![image](https://github.com/Regression-Games/RegressionDocs/assets/112959031/85f60e76-3fcb-4171-8234-8d557e69bf31)